src/main/webapp/edit-recipe.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Edit Recipe</title>
+  <script type="text/javascript" src="script.js"></script>
+</head>
+
+<body>
+  <h1>Edit Recipe!</h1>
+
+  <form action="/new-recipe" method="POST">
+    <h2>Name</h2>
+    <label for="name">What's your recipe called?</label><br/>
+    <textarea id="name" name="name" rows="1"></textarea><br/><br/>
+
+    <h2>Description</h2>
+    <label for="description">Give your recipe some more context.</label><br/>
+    <textarea id="description" name="description" rows="3"></textarea><br/><br/>
+
+    <h2>Tags</h2>
+    <div id="tags">
+      <label for="tag1">Tag 1</label></br>
+      <textarea id="tag1" name="tag1" rows="1"></textarea></br></br>
+      <label for="tag2">Tag 2</label></br>
+      <textarea id="tag2" name="tag2" rows="1"></textarea></br></br>
+    </div>
+
+    <h2>Ingredients</h2>
+    <div id="ingredients">
+      <label for="ingredient1">Ingredient 1</label></br>
```

Comment:
Are you planning to edit this to let the user add more tags, steps and ingredients in the future, possibly by add more button?

Reply:
Yes! I've already coded this functionality, and I'm holding onto its branches in my local repo. I'm doing so because these branches all depended on my previous 4 or 5, and while I was still figuring out how to rebase everything to only show new commits, I messed something up. When I then tried to make their PRs, I had a catastrophe of duplicated commits and other rebasing issues, so I plan on creating their PRs once this PR and the previous once are merged into main-add-recipes, which should allow me to make one clean rebase.